forms/review.form.php

```diff
@@ -1,58 +1,90 @@
 <?php
+//display errors so we know if there is a problem
 ini_set('display_errors', 1);
 
+// we need the User and Review classes
 require_once 'src/User.php';
 require_once 'src/Review.php';
 
+// initialize empty variables
 $comment = '';
 $score = '';
 $userEmail = '';
 $userId = 0;
 
+// set an empty error array
 $error = array();
 
+// check to see if the submit button was clicked on review form
 if (isset($_POST['reviewSubmit'])) {
 	// debugging, leave in for now
-	// echo '<pre>';
-	// var_dump($_POST);
-	// echo '</pre>';
+	echo '<pre>';
+	var_dump($_POST);
+	echo '</pre>';
+
+	// make sure comment isn't empty
 	if (!empty($_POST['comment'])) {
+
```

Comment:
remove empty lines after the if statements use find/replace with regex: `\{\n\n` -> `{\n`

Reply:
Fixed with commit 50025c84eca72bbf8edbb46b37cc308d21089056